app-frontend/src/app/pages/projects/edit/browse/browse.controller.js

```diff
@@ -470,4 +482,12 @@ export default class ProjectAddScenesBrowseController {
         this.setSelected(this.activeScene, !this.isSelected(this.activeScene));
         this.closeDetailPane();
     }
+
+    isInProject(scene) {
+        if (this.projectSceneIds) {
```

Comment:
This works but it might be good to make it use `projectScenesReady` just so the logic is consistent across the whole component.